### PR TITLE
[ChatQnA] Add Nginx in Docker Compose and README

### PR DESCRIPTION
**Pull request by letonghan**
## Description

Add Nginx in Docker Compose and README

## Issues

n/a

## Type of change

List the type of change like below. Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds new functionality)
- [ ] Breaking change (fix or feature that would break existing design and interface)
- [x] Others (enhancement, documentation, validation, etc.)

## Dependencies

None

## Tests

CI test

### CODE REVIEW

File: ChatQnA/docker_compose/intel/cpu/xeon/README.md [hunk 1]
```diff
@@ -216,6 +216,13 @@ cd GenAIExamples/ChatQnA/ui
...
 ```
 
+### 9. Build Nginx Docker Image
+
+```bash
+cd GenAIComps
+docker build -t opea/nginx:latest --build-arg https_proxy=$https_proxy --build-arg http_proxy=$http_proxy -f comps/nginx/Dockerfile .
+```
+
 Then run the command `docker images`, you will have the following 7 Docker Images:
```
Comment (lvliang-intel) [inline ChatQnA/docker_compose/intel/cpu/xeon/README.md:226]: 8 docker images

Comment (letonghan) [reply ChatQnA/docker_compose/intel/cpu/xeon/README.md]: updated

**Comment by kevinintel** [comment]
docker is not published yet

**Comment by letonghan** [comment]
> nginx docker is not published yet

Yes. This PR only updateed inner README to build Nginx docker image from source.
The outer README will be updated with nginx after the docker published.

**Review by lvliang-intel — APPROVED** [review]

**Review by lkk12014402 — APPROVED** [review]

**Pull request merged**